src/kbc_modules/offline_fs_kbc/README.md

```diff
@@ -27,20 +27,30 @@ Resources must be provide in the guest file system at `/etc/aa-offline_fs_kbc-re
 {
   "Policy": "<base64-encoded content from policy.json>",
   "Sigstore Config": "<base64-encoded content from sigstore_config.yaml>",
-  "GPG Keyring": "<base64-encoded content from pubkey.gpg>"
+  "GPG Keyring": "<base64-encoded content from pubkey.gpg>",
+  "Cosign Key": "<base64-encoded content from cosign.gpg>",
+  "Credential": "<base64-encoded content from auth.json>"
 }
 ```
 The values are base64-encoded related file content, can be generated by command such as:
 ```bash
 cat </path/to/policy.json> | base64
 cat </path/to/sigstore_config.yaml> | base64
 cat </path/to/pubkey.gpg> | base64
+cat </path/to/cosign.pub> | base64
+cat </path/to/auth.json> | base64
 ```
 Here is a sample resource file [`aa-offline_fs_kbc-resources.json`](aa-offline_fs_kbc-resources.json) which base on the files under sample_kbc folder:
 - [`policy.json`](../sample_kbc/policy.json)
 - [`sigstore_config.yaml`](../sample_kbc/sigstore_config.yaml)
 - [`pubkey.gpg`](../sample_kbc/pubkey.gpg)
+- [`cosign.gpg`](../sample_kbc/cosign.pub)
+- [`auth.json`](../sample_kbc/auth.json)
+    
+    Referrers
 
+      https://github.com/containers/image/blob/main/docs/containers-auth.json.5.md
+      https://github.com/Xynnn007/image-rs/blob/feat-auth/docs/image_auth.md
```

Comment:
Please wait for https://github.com/confidential-containers/image-rs/pull/71 to be merged and change the private repo link `https://github.com/Xynnn007/image-rs/blob/feat-auth/docs/image_auth.md` to `https://github.com/confidential-containers/image-rs/blob/main/docs/image_auth.md`